support-frontend/assets/components/thankYou/whatNext/whatNext.tsx

```diff
@@ -1,92 +1,61 @@
-}: WhatNextProps): JSX.Element {
-	if (observerPrint) {
-		const observerPrintItem =
+		return <OrderedList items={printProductItems} />;
+	}
+
+	if (observerPrint ?? isGuardianPrint) {
+		const observerItem =
 			'The Observer team will be in touch shortly via email to welcome you. Check your spam folder or add the observer.co.uk domain to your preferred sender list.';
-		const observerSubscriptionCardItems = [
+
+		const subscriptionCardItems = [
 			'You should receive an email confirming the details of your subscription, and another email shortly afterwards that contains details of how you can pick up your newspapers from tomorrow.',
 			'You will receive your Subscription Card in your subscriber pack in the post, along with your home delivery letter.',
 			'Visit your chosen participating newsagent to pick up your newspaper using your Subscription Card, or arrange a home delivery using your delivery letter.',
-			observerPrintItem,
 		];
-		const observerPaperItems = [
+
+		const paperItems = [
 			'Look out for an email from us confirming your subscription. It has everything you need to know about how to manage it in the future.',
 			'Your newspaper will be delivered to your door.',
-			observerPrintItem,
 		];
-		const observerListItems =
-			observerPrint === ObserverPrint.Paper
-				? observerPaperItems
-				: observerSubscriptionCardItems;
-		return <OrderedList items={observerListItems} />;
-	}
-
-	if (isGuardianWeekly) {
-		const myAccountLink = (
-			<a
-				css={neutralFontColor}
-				href={manageSubsUrl}
-				onClick={sendTrackingEventsOnClick({
-					id: 'checkout_my_account',
-					product: 'Paper',
-					componentType: 'ACQUISITIONS_BUTTON',
-				})}
-			>
-				Manage my account
-			</a>
-		);
 
-		const helpCenterLink = (
-			<a
-				css={neutralFontColor}
-				href={helpCentreUrl}
-				onClick={sendTrackingEventsOnClick({
-					id: 'checkout_help_centre',
-					product: 'Paper',
-					componentType: 'ACQUISITIONS_BUTTON',
-				})}
-			>
-				Help centre
-			</a>
-		);
+		const baseItems = isSubscriptionCard ? subscriptionCardItems : paperItems;
+		const allItems = [...baseItems, observerPrint && observerItem];
 
-		const printProductItems = [
-			'Look out for an email from us confirming your subscription. It has everything you need to know about how to manage it in the future.',
-			'Your magazine will be delivered to your door. Please allow 1 to 7 days after publication date for your magazine to arrive, depending on national post services.',
-			<>
-				You can manage your subscription by visiting {myAccountLink}. For any
-				other queries please visit the {helpCenterLink}.
-			</>,
-		];
-
-		return <OrderedList items={printProductItems} />;
+		return <OrderedList items={allItems.filter(Boolean)} />;
 	}
 
-	const listItems = [
+	const fallbackItems = [
 		'You will receive an email confirming the details of your subscription',
 		startDate
 			? `Your payment of £${amount}/month will be taken on ${startDate}`
```

Comment:
Do we ever hit this code path? It's suprising that we're hardcoding `£` here isn't it?